types/express-list-endpoints/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Why disable all these lint rules?

Reply:
@amcasey I copied the rules from a neighboring and very similar types package when I ran into an issue while running `npm run lint express-list-endpoints`; after making some changes to the files and reviewing again, I'm left with just an "npm-naming" issue:

```
Error: <...>/DefinitelyTyped/types/express-list-endpoints/index.d.ts:1:1
ERROR: 1:1  npm-naming  d.ts file must have a matching npm package.
To resolve this error, either:
1. Change the name to match an npm package.
```

The necessary header is included and the express-list-endpoints module [exists on npm](https://www.npmjs.com/package/express-list-endpoints)... is there something I'm overlooking?